src/track/text-style.js

```diff
@@ -148,6 +148,7 @@ class TextStyle {
     let textStyle = new TextStyle();
     textStyle.fontEdge = getValue(setting.fontEdge, textStyle.fontEdge);
     textStyle.fontSize = getValue(setting.fontSize, textStyle.fontSize);
+    textStyle.fontScale = getValue(setting.fontScale, textStyle.fontScale);
```

Comment:
so if app overrides the `fontSize` only, the `fontScale` wiill override it and it won't be changed no?

Reply:
In that case, getValue will receive settings.fontScale=undefined, and will fallback to textStyle.fontScale (which was set by fontSize)

Reply:
great. do we have unit tests for both fontSize and fontScale overrriding?

Reply:
Yes, see my changes in player.spec.js

Reply:
Yes, see my changes to player.spec.js